### PR TITLE
XEP-0369: fix typo error

### PR DESCRIPTION
**Pull request by manuel-rubio**
I've changed the apostrophes in the XML code and add two of them missing.

### CODE REVIEW

File: xep-0369.xml [hunk 1]
```diff
@@ -1314,7 +1314,7 @@ This approach  enables flexible support of multiple clients for a MIX channel pa
     to='hag66@shakespeare.example'
     id='E6E10350-76CF-40C6-B91B-1EA08C332FC7'>
   <leave xmlns='urn:xmpp:mix:1'
-          channel=`coven@mix.shakespeare.example`/>
+          channel='coven@mix.shakespeare.example`/>
```
Comment (ralphm) [inline xep-0369.xml]: You forgot the end quote.

Comment (manuel-rubio) [reply xep-0369.xml]: oh! boy :-P

**Comment by horazont** [comment]
@stevekille Same question as in #607.

**Comment by stevekille** [comment]
Ditto

**Comment by manuel-rubio** [comment]
@horazont @stevekille added a new commit with revision history section.

**Pull request merged**